src/thunderbots_msgs/msg/PlayInfo.msg

```diff
@@ -0,0 +1,8 @@
+# This message contains information about the current Play and Tactics our AI is running
+
+# the name of this team
+string play_type
+
+string play_name
+
+string[] robot_tactic_assignment
```

Comment:
Based on implementation, I should expect a string here of format `"Robot X - Tactic A"`?

Does the string at index 0 correspond to the tactic for robot 0 and so on?

Reply:
Not blocking, just asking for clarification

Reply:
Yes, strings will be in the form `"Robot X - Tactic A"`.

The strings are sorted in order of ascending robot id. So it may be something like:
```
[
    "Robot 0 - Move Tactic",
    "Robot 1 - Move Tactic",
    "Robot 3 - Move Tactic",
    "Robot 4 - Move Tactic",
    "Robot 7 - Move Tactic",
]
```

We don't send a string for robots without a tactic (eg. the robot isn't even on the field).